server/src/termdb.gdc.js

```diff
@@ -715,15 +720,15 @@ async function getOpenProjects(ds) {
 
 	ds.gdcOpenProjects = new Set()
 
-	if (re?.data?.aggregations?.['cases.project.project_id']?.buckets) {
-		for (const b of re.data.aggregations['cases.project.project_id'].buckets) {
-			// key is project_id value
-			if (b.key) ds.gdcOpenProjects.add(b.key)
-		}
-		console.log('GDC open-access projects:', ds.gdcOpenProjects.size)
-	} else {
-		console.log("getting open project_id but return is not re.data.aggregations['cases.project.project_id'].buckets")
+	if (!Array.isArray(re?.data?.aggregations?.['cases.project.project_id']?.buckets)) {
+		console.log("getting open project_id but return is not re.data.aggregations['cases.project.project_id'].buckets[]")
```

Comment:
the message is not very clear.

Reply:
it's just a complain for broken data which should not happen